x-pack/plugins/kubernetes_security/public/components/tree_view_container/breadcrumb/index.tsx

```diff
@@ -6,38 +6,45 @@
  */
 
 import React, { useCallback } from 'react';
-import { EuiButtonEmpty, EuiIcon, EuiToolTip } from '@elastic/eui';
-import { KubernetesCollectionMap, KubernetesCollection } from '../../../types';
+import { EuiButtonEmpty, EuiButtonIcon, EuiIcon, EuiToolTip } from '@elastic/eui';
+import { startCase } from 'lodash';
+import { useEuiTheme } from '../../../hooks';
+import {
+  KubernetesCollectionMap,
+  KubernetesCollection,
+  TreeViewIconProps,
+  KubernetesTreeViewLevels,
+} from '../../../types';
 import { useStyles } from './styles';
-import { TreeViewIcon } from '../tree_view_icon';
 import { KUBERNETES_COLLECTION_ICONS_PROPS } from '../helpers';
-
+import { showBreadcrumbDisplayText } from './helper';
 interface BreadcrumbDeps {
   treeNavSelection: Partial<KubernetesCollectionMap>;
   onSelect: (selection: Partial<KubernetesCollectionMap>) => void;
 }
 
 export const Breadcrumb = ({ treeNavSelection, onSelect }: BreadcrumbDeps) => {
   const styles = useStyles();
+  const { euiVars } = useEuiTheme();
   const onBreadCrumbClick = useCallback(
     (collectionType: KubernetesCollection) => {
```

Comment:
Interesting that TS is not complaining about this, shouldn't the type now be changed to `KubernetesTreeViewLevels`?

Reply:
Hmm why would typescript complain abouteuiVars. `euiVars` looks like collection object of elastic theme colors.  
<img width="676" alt="image" src="https://user-images.githubusercontent.com/17135495/207166291-828233f8-80b7-437c-9bba-98217fb9235a.png">

KubernetesTreeViewLevels is an enum that the different types of cluster resouce levels: Cluster, Node, Pod, and Container Image.

Reply:
pointed the Github line wrong, I was referring to the useCallback parameter `collectionType` being `KubernetesCollection`

```js
const onBreadCrumbClick = useCallback(
    (collectionType: KubernetesCollection) => {
```